src/postgres/getPostgresTablesQueryRows.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientConfig } from "pg";
+import { runPostgresQuery, wrapArgInQuotes } from "./runPostgresQuery";
+
+export interface IPostgresTable {
+    schemaName: string;
+    name: string;
+    oid: number;
+    columnNames: string[];
+}
+
+function getTableInfo(): string {
```

Comment:
I think its worthwhile to combine these three queries into one to make it more efficient. Something like this would be a good starting point:

```sql
select schemaname,
tablename,
pg_class.oid,
columns.column_name

from pg_catalog.pg_tables
left join pg_class on relname = tablename
left join information_schema.columns columns on table_name = tablename

where schemaname != 'pg_catalog' AND
schemaname != 'information_schema'
```

But I'm not sure if table names are unique so it would be worth looking into that & running the query through some testing